qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/LocationManagementSteps.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openmrs.contrib.qaframework.RunTest;
+
+import org.openmrs.contrib.qaframework.page.ManageLocationAttributeTypesPage;
+import org.openmrs.contrib.qaframework.page.LocationPage;
+import org.openmrs.contrib.qaframework.page.ManageLocationsPage;
+import org.openmrs.contrib.qaframework.page.AddEditLocationPage;
+import org.openmrs.contrib.qaframework.page.ManageLocationTagsPage;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+import org.openqa.selenium.By;
+
+public class LocationManagementSteps extends Steps {
+    private static final String TAG_NAME = "This is a tag name";
+    private static final String MAX_VALUE = "20";
+    private static final String MIN_VALUE = "1";
+    private static final String DESCRIPTION = "This is the description";
+    private static final String NAME = "Noah";
+    private static final String LOCATION_NAME = "Bugema";
+    private static final String ADDRESS_ONE = "Bugema one";
+    private static final String ADDRESS_TWO = "Bugema two";
+    private static final String CITY = "Kampala";
+    private static final String STATE = "Central";
+    private static final String COUNTRY = "Uganda";
+    private ConfigureMetadataPage configureMetadataPage;
+    private LocationPage locationPage;
+    private ManageLocationsPage manageLocationsPage;
+    private AddEditLocationPage addEditLocationPage;
+    private ManageLocationTagsPage manageLocationTagsPage;
+    private ManageLocationAttributeTypesPage manageLocationAttributeTypesPage;
+
+    @Before(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void destroy() {
+        quit();
+    }
+
+    @Given("a user clicks on the configure metadata link from the home page")
+    public void clicksConfigureMetaDataLink(){
+        configureMetadataPage = homePage.goToConfigureMetadata();
+    }
+
+    @When("the system loads configure metadata link dashboard")
+    public void loadsConfigureMetaData(){
```

Comment:
```suggestion
    public void loadsConfigureMetaData() {
```
please do the same for all the other methods